style.css

```diff
@@ -562,6 +661,69 @@ span {
   display: none;
 }
 
+.pop-header {
+  display: flex;
+  width: 100%;
+}
+
+.pop-header-div {
+  margin-top: 27px;
+  margin-right: 30px;
+}
+
+.pop-header-h3 {
+  margin-left: 16px;
+  margin-right: 200px;
+  margin-top: 12px;
+  margin-bottom: 0;
+  font-weight: bold;
+  font-size: 32px;
+  color: #172b4d;
+  font-family: Poppins, sans-serif;
+  line-height: 44px;
+}
```

Comment:
- [ ] Still in the path of achieving the Figma design match, Kindly ensure that the `font-size` is changed to `40px` as it is provided in the Figma design. This will help us to make our portfolio professional and highly attractive to users ☺️.

Reply:
Thank you for letting me know. the issue has been rectified by fixing the font size.